sdk/storage/azure-storage-file-share/azure/storage/fileshare/_shared/models.py

```diff
@@ -333,17 +333,17 @@ class Services(object):
         Access for the `~azure.storage.blob.BlobServiceClient`
     :param bool queue:
         Access for the `~azure.storage.queue.QueueServiceClient`
-    :param bool file:
-        Access for the `~azure.storage.file.FileServiceClient`
+    :param bool fileshare:
+        Access for the `~azure.storage.fileshare.ShareServiceClient`
```

Comment:
Should the changes in this shared file be carried over to blobs and queues?

Reply:
Yes - they should :)